<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FIS</class>
 <widget class="QMainWindow" name="FIS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fingerprint Image Segmentation</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resource/img/fingerprint.png</normaloff>:/resource/img/fingerprint.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Import Image</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="originalimage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>301</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Original </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Segmentation method</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply Segmentation</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioSeg1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Simple Segmentation</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioSeg2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>450</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Simple Segmentation 2</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="initseg">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>301</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="post">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>310</y>
      <width>301</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>550</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Initial segmentation</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>550</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Post-processing</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>801</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>20</width>
      <height>251</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="reference">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>150</width>
      <height>115</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Import Reference</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>240</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Calc. Comparison Score</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>199</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioSeg2_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Harris Corner Point</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="comparison">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>70</y>
      <width>150</width>
      <height>115</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Reference Image</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>190</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Segmented Comparison</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>833</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
